.rubocop.yml

```diff
@@ -25,13 +25,20 @@ Metrics/BlockLength:
     - config/**/*
     - spec/**/*
 
+Metrics/MethodLength:
+  Exclude:
+    - spec/**/*
+
 Lint/AmbiguousBlockAssociation:
   Exclude:
     - spec/**/*
 
 Style/Documentation:
   Enabled: false
 
+Style/RedundantInterpolation:
```

Comment:
Is this rule needed to be disabled? I think it can be useful.

Reply:
I found this useful for keeping consistency
Without that we could not use `"#{foobar}"` version becasue rubocop screams about that and I think it is bad because if we want add more to the string e.g. `"#{foobar} is a buzzword"` looks easy to change but with
`foobar`.to_s we have to do `foobar.to_s + "is a buzzword"` or with "<<" operator.
Adding strings in Ruby is slow.. :( 

Sum up: Removing this rule let me use version "#{foobar}" without screaming on me and I think it is okay